src/Modules/Engine.cpp

```diff
@@ -56,6 +56,10 @@ Variable sar_tick_debug("sar_tick_debug", "0", 0, 3, "Output debugging informati
 
 Variable sar_cm_rightwarp("sar_cm_rightwarp", "0", "Fix CM wrongwarp.\n");
 
+float g_cur_fps = 0.0f;
+static uint64_t g_total_frames = 0;
```

Comment:
bad name since it's total frames *this second*, and also should probably be function-static (see next comment)